beacon-chain/sync/metrics.go

```diff
@@ -69,6 +69,13 @@ var (
 			Help: "Count the number of times attestation not recovered and pruned because of missing block",
 		},
 	)
+	arrivalBlockPropagationHistogram = promauto.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "block_arrival_latency_milliseconds",
+			Help:    "Captures blocks propagation time. Blocks arrival in milliseconds distribution",
+			Buckets: []float64{1, 2, 3, 4, 5, 6},
```

Comment:
Do you need to update these buckets to be milliseconds?

Reply:
Ah yes. Nice catch. I just fixed it.

Really need ☕ ....